test/common/index.js

```diff
@@ -883,6 +885,7 @@ const common = {
   hasIntl,
   hasCrypto,
   hasQuic,
+  hasSQLite,
```

Comment:
Can we move the declaration here?

Reply:
@anonrig can you clarify what you mean on this one?

Reply:
That is not consistent with how any of the other ones are done as far as I can see. I was just trying to follow what seems to be existing practice.